python/test/integration/test_histogram.py

```diff
@@ -73,12 +72,7 @@ def test_count_by_threshold():
     assert pre(data) == {"a": 500, "b": 200, "other": 100}
     print("laplace_histogram_from_dataframe", laplace_histogram_from_dataframe(data))
     assert scale == 1.0
-    assert threshold == 18.727533563392424
-
-    with pytest.raises(dp.OpenDPException):
-        dp.m.make_laplace_threshold(
-            dp.atom_domain(T=int), dp.l1_distance(T=float), scale=1.0, threshold=1e8
-        )
```

Comment:
Would like to keep an assertion on this expression: If it's no longer an error, what is it?